.github/workflows/ci_push_testing.yml

```diff
@@ -0,0 +1,223 @@
+        source ~/dirac_ui/bashrc
+        dirac-proxy-init -x
+        dirac-configure -F -S GridPP -C dips://dirac01.grid.hep.ph.ic.ac.uk:9135/Configuration/Server -I
+        dirac-proxy-init -g gridpp_user -M
+    - name: Install gangarc file
+      run: |
+        echo -e "[DIRAC]\nDiracEnvSource = ~/dirac_ui/bashrc" > ~/.gangarc
+        echo -e "[Configuration]\nRUNTIME_PATH=GangaDirac" >> ~/.gangarc
+        echo -e "[defaults_DiracProxy]\ngroup=gridpp_user" >> ~/.gangarc
+    - name: Test with pytest
+      env:
+        GANGA_CONFIG_FILE: ~/.gangarc
+        GANGA_CONFIG_PATH: GangaDirac/Dirac.ini
+        DIRAC_DEPRECATED_FAIL: True
+      run: python3 -m pytest --cov-report term-missing --cov ganga/GangaDirac ganga/GangaDirac/test/GPI
+
+  lhcb-unit:
+    name: GangaLHCb Unit
+    needs: lint
+    runs-on: ubuntu-latest
+    container: 
+      image: centos:7
+      options: --privileged
+    steps:
+    - name: Install CVM-FS
+      run: |
+        yum install -y https://ecsft.cern.ch/dist/cvmfs/cvmfs-release/cvmfs-release-latest.noarch.rpm
+        yum install -y cvmfs cvmfs-config-default
+        cvmfs_config setup
+        echo "CVMFS_REPOSITORIES=lhcb.cern.ch,sft.cern.ch" > /etc/cvmfs/default.local
+        echo "CVMFS_HTTP_PROXY=DIRECT" >> /etc/cvmfs/default.local
+        mkdir -p /cvmfs/lhcb.cern.ch
+        mkdir -p /cvmfs/sft.cern.ch
+        mount -t cvmfs lhcb.cern.ch /cvmfs/lhcb.cern.ch
+        mount -t cvmfs sft.cern.ch /cvmfs/sft.cern.ch
+    - uses: actions/checkout@v1
+    - name: Install virtualenv
+      run: |
+        source /cvmfs/lhcb.cern.ch/lib/LbEnv && source LbLogin.sh
+        /cvmfs/sft.cern.ch/lcg/releases/LCG_97python3/Python/3.7.6/x86_64-centos7-gcc9-opt/bin/python3 -m venv ~/venv
+    - name: Install dependencies
+      run: |
+        . ~/venv/bin/activate
+        python3 -m pip install --upgrade pip setuptools wheel
+        python3 -m pip install -e .[LHCb]
+        python3 -m pip install coverage pytest pytest-cov pytest-pylint pytest-mock
+    - name: Test with pytest
+      run: |
+        . ~/venv/bin/activate
+        python3 -m pytest --cov-report term-missing --cov ganga/GangaLHCb ganga/GangaLHCb/test/Unit
+  
+  fuse-test:
+    name: Fuse Test
+    runs-on: ubuntu-latest
+    container: 
+      image: centos:7
+      options: --privileged
+    steps:
+    - name: Install CVM-FS
+      run: |
+        yum install -y https://ecsft.cern.ch/dist/cvmfs/cvmfs-release/cvmfs-release-latest.noarch.rpm
+        yum install -y cvmfs cvmfs-config-default
+        cvmfs_config setup
+        echo "CVMFS_REPOSITORIES=lhcb.cern.ch" > /etc/cvmfs/default.local
+        echo "CVMFS_HTTP_PROXY=DIRECT" >> /etc/cvmfs/default.local
+        mkdir -p /cvmfs/lhcb.cern.ch
+        mount -t cvmfs lhcb.cern.ch /cvmfs/lhcb.cern.ch
+    - name: Test CVM-FS
+      run: ls /cvmfs/lhcb.cern.ch
+    - name: Load LHCb environment
+      run: source /cvmfs/lhcb.cern.ch/lib/LbEnv
```

Comment:
Guess that this can just be deleted now.

Reply:
Yes indeed. I will do that today as well as adding the LHCb integration tests.